guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,192 @@
+
+```json
+[
+  {
+      "id": "495925",
+      "title": "Doraemon the Movie:Nobita's Treasure Island",
+      "director": "Fujiko Fujio",
+      "genre": "Science fiction",
+      "poster": "https://image.tmdb.org/t/p/w1280/cmJ71gdZxCqkMUvGwWgSg3MK7pC.jpg",
+      "overview": "The story is based on Robert Louis Stevenson's Treasure Island novel.",
+      "release_date": 1520035200
+  },
+  {
+      "id": "329996",
+      "title": "Dumbo",
+      "director": "Tim Burton",
+      "genre": "Science fiction",
+      "poster": "https://image.tmdb.org/t/p/w1280/279PwJAcelI4VuBtdzrZASqDPQr.jpg",
+      "overview": "A young elephant, whose oversized ears enable him to fly, helps...",
+      "release_date": 1553644800
+  },
+  {
+      "id": "458723",
+      "title": "Us",
+      "director": "Jordan Peele",
+      "genre": "Horror",
+      "poster": "https://image.tmdb.org/t/p/w1280/ux2dU1jQ2ACIMShzB3yP93Udpzc.jpg",
+      "overview": "Husband and wife Gabe and Adelaide Wilson take their...",
+      "release_date": 1552521600
+  },
+  ...
+]
+```
+
+You would declare faceted attributes as follows:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "attributesForFaceting": [
+          "director",
+          "genre"
+      ]
+  }'
+```
+
+## Querying On Faceted Attributes
+
+When performing a search, you can pass parameter to your query:
+
+### 1. The facets to filter on
+
+You can filter on facets to narrow down your results based on criteria.
+
+`facetFilters=["facetName:facetValue"]`, `facetFilters=[["facetName:facetValue"]]` or a mix of both `facetFilters=["facetName1:facetValue1", ["facetName2:facetValue2"]]`
+
+- `["facetName1:facetValue1", ["facetName2:facetValue2"]]` (Array of strings or arrays of strings, defaults to `null`)
+
+  Both types of array contain the facet names and values to filter on.
+  A valid array must be an array which contains either a list of strings or arrays of strings and can mix both (e.g. `["kind:t-shirt", ["color:red", "color:green"]]`).
+
+  - `facetName`: The name (the attribute) of a field used as a facet (e.g. `color`, `kind`).
+  - `facetValue`: The value of this facet to filter results on (e.g. `red`, `green`, `t-shirt`, `pants`).
+
+### Example
+
+Given a clothing dataset, suppose you want to retrieve all t-shirts with a nautical pattern. You would then use:
+
+```bash
+$ curl --get 'http://localhost:7700/indexes/clothing/search' \
+    --data-urlencode 'q=nautical' \
+    --data-urlencode 'facetFilters=["kind:t-shirt"]'
+```
+
+### Logical Connectives
+
+Inputting a double dimensional array allows you to use **logical connectives**.
+
+- **Inner arrays elements** are connected by an `OR` operator (e.g. `[["color:red", "color:green"]]`).
+- **Outer arrays elements** are connected by an `AND` operator (e.g. `["color:red", "kind:t-shirt"]`).
+
+You can mix connectives, for instance, the following array:
+
```

Comment:
not how I imagined it, but it turns out pretty good, GJ!

Reply:
Thanks! :smile:

Reply:
Thanks! Don't hesitate to tell me if you want to modify/add things! 😄